TombEngine/Objects/TR3/Entity/tr3_shiva.cpp

```diff
@@ -305,12 +336,12 @@ namespace TEN::Entities::Creatures::TR3
 					if (TestProbability(0.0325f))
 						item->Animation.TargetState = SHIVA_STATE_WALK_FORWARD;
 				}
-				else if (AI.bite && AI.distance < pow(SECTOR(1.25f), 2))
+				else if (AI.bite && AI.distance < SHIVA_GRAB_ATTACK_RANGE)
 				{
 					item->Animation.TargetState = SHIVA_STATE_GRAB_ATTACK;
 					creature->Flags = 0;
 				}
-				else if (AI.bite && AI.distance < pow(SECTOR(4) / 3, 2))
+				else if (AI.bite && AI.distance < SHIVA_DOWNWARD_ATTACK_RANGE)
```

Comment:
It seems that `SHIVA_DOWNWARD_ATTACK_RANGE` is decreased from 1.33 blocks to 1 block. Is this intentional?

Reply:
Will fix it then.

Reply:
Fixed